skfp/descriptors/topological.py

```diff
@@ -218,9 +404,46 @@ def zagreb_index(mol: Mol) -> int:
     Examples
     --------
     >>> from rdkit.Chem import MolFromSmiles
-    >>> from skfp.descriptors.topological import zagreb_index
+    >>> from skfp.descriptors.topological import zagreb_index_m1
     >>> mol = MolFromSmiles("C1=CC=CC=C1")  # Benzene
-    >>> zagreb_index(mol)
+    >>> zagreb_index_m1(mol)
     24
     """
     return int(sum(atom.GetDegree() ** 2 for atom in mol.GetAtoms()))
+
+
+def zagreb_index_m2(mol: Mol) -> int:
+    """
+    Second Zagreb Index.
+
+    Calculates the Second Zagreb Index [1]_, defined as the sum of the product of
```

Comment:
`second Zagreb index` would suffice, it's not really a specific name to use capitalized words